mod/layout/set/abstract/thumbnail.rb

```diff
@@ -9,12 +9,12 @@
     end
   end
 
-  view :thumbnail do
+  view :thumbnail do |args|
```

Comment:
The wild voo bull always throws me off when I need custom arguments so I use args instead.